spec/models/reports/daily_follow_up_spec.rb

```diff
@@ -1,165 +0,0 @@
-require "rails_helper"
```

Comment:
Should add similar model specs for the new view.

Reply:
Will add spec for this model